src/mixins/bonds.js

```diff
@@ -123,7 +147,8 @@ export const BondsMixin = (superclass) => class extends superclass {
     }
 
     /**
-     * Iterates over all combination of atoms and draws bonds.
+     * Draw bonds. Bonds are created synchronously if the asynchronous callback (createBondsAsync) to draw bonds
+     * in background has not returned yet. This may happen if the structure is large and draw bonds is toggled quickly.
```

Comment:
We need this to block the UI until the bonds are drawn.

Reply:
Done.